spec/abci++/v0.md

```diff
@@ -0,0 +1,156 @@
+# Tendermint v0 Markdown pseudocode
+
+This translates the latex code for Tendermint consensus from the Tendermint paper into markdown.
+
+### Initialization
+
+```go
+h_p ← 0
+round_p ← 0
+step_p is one of {propose, prevote, precommit}
+decision_p ← nil
```

Comment:
In the Tendermint paper, `decision_p` is a dynamically growing vector of values. I guess, that is why the authors wrote `decision_p[] := nil` in the paper. Can we write:

```
decision_p ← Vector()
```

Or `Array()`, or `List()`. I am fine with any choice here.

Reply:
Set to vector for now, but I wonder if for technical correctness we should actually put `Map`, since its indexed by `h_p`, and we don't initialize at 0